elastic/apm-server
# model: move Event, Session, and URL up to APMEvent

## Motivation/summary

Also, remove Transaction.Page and Error.Page. Page details have been duplicated in ECS fields since 7.9.0, and we don't reference the page fields in the UI.

Requires https://github.com/elastic/kibana/pull/107613

## Checklist

~- [ ] Documentation has been updated~

## How to test these changes

1. Send some RUM page-load transactions
2. Compared to how they appear in the UI with 7.14; `transaction.page.*` fields should no longer be stored, but there should be no change otherwise

## Related issues

#4120
#3565
